### PR TITLE
[Validator] Add missing Hungarian validator translations

### LINKED ISSUE #51943: Missing translations for Hungarian (hu)
Hello,

There are some missing translation for Hungarian (hu). These should be added in branch 5.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf)

> [!NOTE]
> If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by 1ed**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no
| Issues        | Fix #51943
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf [hunk 1]
```diff
@@ -402,6 +402,30 @@
...
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>A fájlnév túl hosszú. {{ filename_max_length }} karakter vagy kevesebb legyen.|A fájlnév túl hosszú. {{ filename_max_length }} karakter vagy kevesebb legyen.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>The password strength is too low. Please use a stronger password.</source>
+                <target>A jelszó túl egyszerű. Kérjük, használjon egy bonyolultabb jelszót.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>Ez az érték olyan karaktereket tratalmaz, amik nem megengedettek.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf:415]: Google Translate translates this to: `This value treats characters that are not allowed.`. Can you double check that this is correct?

File: src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf [hunk 2]
```diff
@@ -402,6 +402,30 @@
...
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Using invisible characters is not allowed.</source>
+                <target>Láthatatlan karaktert használata nem megengedett.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Mixing numbers from different scripts is not allowed.</source>
+                <target>Különböző szám írásmódok használata nem megengedett.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf:423]: Google translate says: 

`The use of different spellings of numbers is not allowed.`

Can you please double check this too?

**Review by Nyholm — COMMENTED** [review]
Great. Thank you very much. 

I do not speak Hungarian so I tried my best to verify this with Google translate. I know it is a pretty bad tool... But can you just double check your translations on two places? (they might be good already)

File: src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf (hunk 1 above)
Comment (1ed) [reply src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf:415]: Correctly, this should be translated to: "This value contains characters that are not allowed." I left out the "current restriction-level" part as in the translation it makes no sense to me and seems not relevant.

File: src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf (hunk 2 above)
Comment (1ed) [reply src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf:423]: I think this is ok, "írásmód" means "way of writing" like "Cyrillic script" or "Cyrillic writing" -> "Cirill írás"

**Comment by 1ed** [comment]
Hey, thank you for reviewing. I made a research and looked at the relevant PRs and tests to see what are these message about... and tried my best to translate them.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @1ed.

**Pull request merged**